src/site/_data/paths/learn-web-vitals.json

```diff
@@ -0,0 +1,40 @@
+{
+    "slug": "learn-web-vitals",
+    "cover": "/images/collections/web-vitals.svg",
+    "title": "Web Vitals",
+    "description": "Key signals of quality user experiences.",
+    "overview": "Web Vitals define the performance metrics and measurements that indicate a delightful user experience.",
```

Comment:
Probably best to reuse the same language from the web.dev/vitals intro content: "An initiative to provide unified guidance for quality signals that are essential to delivering a great user experience on the web."

Side note: I've always found that language to be a bit awkward, so if we want to change it we should probably change it in both places.